staging/src/k8s.io/client-go/tools/remotecommand/fallback_test.go

```diff
@@ -165,6 +166,77 @@ func TestFallbackClient_SPDYSecondarySucceeds(t *testing.T) {
 	}
 }
 
+// See (https://github.com/kubernetes/kubernetes/issues/126134).
```

Comment:
nit: move this to the end to make cherry-picking easier

Reply:
Done